cob_hardware_config/cob3-2/config/local_costmap_params.yaml

```diff
@@ -22,10 +22,10 @@ inflation_radius: 0.55 #[m]
 
 # sensor setup
 observation_sources: laser_scan_sensor_front laser_scan_sensor_rear laser_scan_sensor_top #point_cloud_sensor_kinect
-laser_scan_sensor_front: {sensor_frame: base_laser_front_link, data_type: LaserScan, topic: scan_front, marking: true, clearing: true}
+laser_scan_sensor_front: {sensor_frame: base_laser_front_link, data_type: LaserScan, topic: base_laser_front/scan_filtered, marking: true, clearing: true}
```

Comment:
should not be *base_laser_front/scan* instead of *base_laser_front/scan_filtered* ?